src/Core/Commander/InterfaceCommander.js

```diff
@@ -13,7 +13,7 @@ function InterfaceCommander(type, priorityFunction) {
     this.type = type;
 
     if (!Date.now) {
-        this.timestamp = function () {
+        this.timestamp = function timestamp() {
```

Comment:
According to https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Date/now Date.now should be available in all navigators we target (including IE >= 9)...
So what about removing the `timestamp` function insteand?